app/views/confirmations/_callback.html.haml

```diff
@@ -0,0 +1,14 @@
+%p
+  We will call you within one working day, between
+  %strong= callback.time
```

Comment:
Alex has pointed out that the callback time also includes the word "between" - perhaps we should fix that repetition in this PR: https://trello.com/c/e1aGvtgs/166-content-tweak

Reply:
Seems sensible to me :+1:

Reply:
Done